botwo/botwo.py

```diff
@@ -1,131 +1,144 @@
 import fnmatch
+import botocore
 
-SPECIAL_METHODS = {"get_paginator", "delete_objects"}
 COPY_METHODS = {"copy", "copy_object", "copy_upload_part"}
+LIST_METHODS = {"list_objects", "list_objects_v2", "list_object_version"}
 
 
-def client(client_mapping, config):
-    botwo = Botwo()
-    return botwo.create_client_class(client_mapping, config)
+class PaginatorWrapper(object):
+    """ This class wraps boto paginator """
```

Comment:
This docstring can be more helpful to the user, and also satisfy PEP-257.  E.g.:
```suggestion
    """Wrapper for a boto paginator.
    
    Holds multiple paginators, one for each client, and dispatches calls to the appropriate
    paginator according to ...
    """
```
("..." needs to be updated to describe what it does).

Reply:
Done